### PR TITLE
DB-related corrections and clarifications

### PR DESCRIPTION
**Pull request by arca0**
Changes in database.md are style corrections and a single clarification to reset from migratus.

Changes in guestbook.md are fixes to what I think might be a bug. In the migration we add the timestamp field, but in save-message! in queries.sql, we don't provide access to that field at all. As a result, the function call from lines 434-436 (save-message! from REPL) does not set the timestamp. On the other hand, the save-message! function we define to handle our form POST does not attempt to set the timestamp (which is technically correct, but doesn't use the timestamp field we defined in db migration).

For clarity, I'll also add comments per line.

I also added Sublime Text to the list of IDEs ;)

### CODE REVIEW

File: resources/md/guestbook.md [hunk 1]
```diff
@@ -416,8 +416,8 @@ denoted using `:` notation. Let's replace the existing queries with some of our
 -- :name save-message! :! :n
 -- :doc creates a new message
 INSERT INTO guestbook
-(name, message)
-VALUES (:name, :message)
+(name, message, timestamp)
```
Comment (arca0) [inline resources/md/guestbook.md:419]: without this, the example call from REPL (below) did not set timestamp properly:
```
(def query-fn (:db.sql/query-fn state/system))

(query-fn :save-message! {:name      "m1"
                          :message   "hello world"
                          :timestamp (java.util.Date.)})
```

File: resources/md/guestbook.md [hunk 2]
```diff
@@ -496,7 +496,7 @@ We'll create a new controller that will be responsible for saving new messages i
           (empty? message)
           (assoc-in [:flash :errors :message] "message is required"))
         (do
-          (query-fn :save-message! {:name name :message message})
+          (query-fn :save-message! {:name name :message message :timestamp (java.util.Date.)})
```
Comment (arca0) [inline resources/md/guestbook.md:499]: Without this, there was no way to set the timestamp through the form's POST operation (after making the previous change).

**Comment by yogthos** [comment]
Timestamp should get set automatically via column property set by the migration [`timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP`](https://github.com/kit-clj/kit-examples/blob/master/guestbook/resources/migrations/20220104235847-add-guestbook-table.up.sql)

**Comment by arca0** [comment]
Yea, that's probably a better way to do this. In the current version of the documentation, our migration does not set the default value of the timestamp though:
```
CREATE TABLE guestbook
(id INTEGER PRIMARY KEY AUTOINCREMENT,
 name VARCHAR(30),
 message VARCHAR(200),
 timestamp TIMESTAMP(7));
```

I'll fix that instead.

**Pull request merged**